app/models/megaphone/podcast.rb

```diff
@@ -22,7 +22,7 @@ class Podcast < Integrations::Base::Show
     # All other attributes we might expect back from the Megaphone API
     # (some documented, others not so much)
     OTHER_ATTRIBUTES = %i[id created_at updated_at image_file uid network_id recurring_import
-      episodes_count spotify_identifier default_ad_settings iheart_identifier feed_url
+      episodes_count spotify_identifier default_ad_settings iheart_identifier feed_url span_opt_in
```

Comment:
getting this attribute back on some podcasts, when listing them, and causes problems if unknown